services/mongodb/compose.yaml

```diff
@@ -5,6 +5,9 @@ services:
       - backend${BE_VERSION:-v4}_mongodb_data:/data/db
       - ./config/init.sh:/docker-entrypoint-initdb.d/init.sh
       - ./config/seed:/seed
+    labels:
+      # Adding a label for traefik to learn which port to look for a service on
```

Comment:
I don't see why we need to open a mongo port through Traefik. Am I missing something?